src/registrar/models/utility/generic_helper.py

```diff
@@ -249,6 +247,7 @@ def _validate_new_instance(self):
                     "Cannot add organization_type and generic_org_type simultaneously "
                     "when generic_org_type, is_election_board, and organization_type values do not match."
                 )
+                logger.error("_validate_new_instance: %s", message)
```

Comment:
These aren't related to the issue, I just added them to help myself understand the code and thought it was worth leaving in.

Reply:
For ease of troubleshooting if this occurs, I would print the values of these fields. Save a future dev the search time.

Reply:
Good idea. Let me know if you like this formatting.

Reply:
Unless I am missing it, I don't see you actually printing the values for those 3, just the message saying they don't match, did you mean to push a different change?

Reply:
Check under the changes tab, it's showing up there but not under the conversation for whatever reason.